designer/server/src/test/scala/pl/touk/nussknacker/ui/api/NuDesignerApiAvailableToExposeYamlSpec.scala

```diff
@@ -164,7 +163,6 @@ object NuDesignerApiAvailableToExpose {
   private def createInstanceOf(clazz: Class[_ <: BaseEndpointDefinitions]) = {
     val basicAuth = auth
       .basic[Option[String]]()
-      .map { AnonymousAccess.optionalStringToAuthCredentialsMapping(false) }
```

Comment:
Well, after looking into it - we need this and a bit more.
I added the mapping for anonymous access and also for impersonation.

Since we have an additional optional header now I also had to udpate the OpenAPI spec file.
Thank you for pointing this one out.